### ISSUE TITLE
[ollama] Error xxxx is not valid. Should be a string, a list/tuple of strings or a list/tuple of integers.

### ISSUE THREAD

**Issue by ednow** [langgenius/dify#8914]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.8.3

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

I tested llava 34B with ollama.However seems issues occured, when it generated a while. 

![image](https://github.com/user-attachments/assets/f6a78f3d-bedb-4950-8667-359f38e30e16)

the error modal detail:

```
[ollama] Error: Input [TextPromptMessageContent(type=<PromptMessageContentType.TEXT: 'text'>, data='描述下这张图'), ImagePromptMessageContent(type=<PromptMessageContentType.IMAGE: 'image'>, data='data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAANYAAACFCAIAAACom5EgAAAGiElEQVR4Ae3cQU7bThTHcS6RY/UIcJVs2kUOwbab7rkA6yKB9FfFoiIitVRBUBDBSjYwf70ZezIxqYkJyc9xvhFCrpPmJe99/GbGxDly3MiANANH0ugEJwMOgiAQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEB6CGBBnAILiAhAeghgQZwCC4gIQHoIYEGcAguICEL6NBPO5G43dr8xdDd3lQf5cDe3tj8Yun3ef6NHr62t8l23Yvr13//1x2YObTN3zzGpwgD/PM3v72YOlYjR2bajL9l5Du7rg77/u5u5w5b092J5nlpDff2OX6OCGdcFwc86Otre3ne2/vbd0vy0De27u3O19tTo7q0uFxKfHbUsXzOc26BzsyFt/mD3PLDldnRe2heBobFOf+koc8r3Zg00KO3lrC8FfmU3ADxlZ/XufTG2N3MlbW1bEV0NG4boj8Hlmp6jCPCxC3N4qNYTYzfO3pQteDusKUN8hDuTeS08w+uvMBgT3hj4Et3vUbdIFr7OX6+ylWS/Mpscnk8GFy/1G72Sy/PN0Hk6JXzz1Th6/Zw2Ynp9Oev3psHpGPR+cTHr9xyJofFrb8C+j+vgVESHYXoLffsy+/Zg1JDgfRgdW/vn3/qR3mlefZOkxbnj2mEodXDgDF/n6/76KYPA3HWb5oD8x02cmz/+Y9WE2r8ZdJRKC3SJYqbHvhcdnSxSWeFkPm54bQd8U0+4VG+rcU4si/UaV6eLeZs01n9ufyzt5W/3XkfQM+G62NxmIP9AFfT97Og/Nb8Gi2tLysguG3vYeQRs9F10wZVoRv7jLum+F/r86YiAY/1axm7qEcFuN1YXlyEcIZrmNvH7SFobXxbwwHY4bE/QDepgLRmeV6abd6x9W0m9EsHuNcC8JXmcvgV34/eXr9MvXabpn/dVJ8LdAUHApliNlsyx62xpd0I/FAfEyQSOe9shKX1zjn50diFtyVDUaiN8l+K+xrLo/KvHTuBLiopPFUXXdgTidU5YdtLro7k8XK6HFC1ixBK682s4S3M0Z8AC9JlYjgpXafGAgThqSgRtcWPc6Pn06LtYcoQt6i8k6t6YLhuXtwBbI5TpjmeBSF/TyPrYirslh7Cb79Zi9HIg/g2DBKwzEvdOn5Lyd52irY9vwdNIHL62IrTuelGdzAqw4lVwm6MH5B6ddcI3xN77ZLnfBHax6wnFZs5rbdRcsRsypyTM0BTsTGRphf/rTTsEstcPCa7mMCI2tIFI0tmISmc/DSUTv1cd62wUXe6LaWpGsiGOb38rGjgl6TOmIaf2pnAsW07LzU9sT2Q0uElUeXEpwaEvs4gmL1hi7Y0HQlJdRfFstKVfixrZX2ehsF9wKqOZPugnB9de/laLu1z8h2JxVk/+xCcH9kvThV3sQBNOVVOpnB/sh+C7NtwR3UJeUQdz+3LhtWRHzkdV6gvEjq9FBZzbaQpAP7tcT7PIH91tyMHH5Uj1BLl/aOlQu4qwhyEWcW/cXAozGXMq++i/FN3edvYLTOVdcQRfWOPLffKFHpRemX+ghr0741OCn/27LciR22tGYrzWy61nTrzWKyenkRusIOmeDEV/udkBf7tbJA4s3tUcZWPFtCumcI7yTdE/YZj95SFVs4qGNA/EeHcG81M0z0JYr6NrwmUVew1avlItds5JnuuDmhzHPsFEG/gcWq63YNgbjYgAAAABJRU5ErkJggg==', detail=<DETAIL.HIGH: 'high'>)] is not valid. Should be a string, a list/tuple of strings or a list/tuple of integers.
```

**ollama version**: 0.3.12
**llava model detail**:
```
ollama show llava:34B
  Model
    architecture        llama
    parameters          34.4B
    context length      4096
    embedding length    7168
    quantization        Q4_0

  Projector
    architecture        clip
    parameters          349.64M
    embedding length    1024
    dimensions          768

  Parameters
    stop    "<|im_start|>"
    stop    "<|im_end|>"

  License
    Apache License
    Version 2.0, January 2004
```

### ✔️ Expected Behavior

llava model functionally generate text 

### ❌ Actual Behavior

stop when llava model generate in a half

**Comment by CQchunqiu** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by crazywoola** [comment]
Will be fixed in 0.9.0

**Comment by ednow** [comment]
> Will be fixed in 0.9.0

When will 0.9.0 be finished, I can't find any clues on this website[road](https://roadmap.dify.ai/roadmap)
